docs/CaliptraIntegrationSpecification.md

```diff
@@ -626,14 +626,15 @@ For additional information, see [Caliptra assets and threats](https://github.com
-| Resets and Clocks                | SoC shall ensure that any programmable Caliptra clock controls are restricted to the SoC Manager.                                                                                                                                                    | Statement of conformance | Required for Caliptra threat model                |
-| Resets and Clocks                | SoC should defend against external clock stop attacks.                                                                                                                                                    | Statement of conformance | Required for Caliptra threat model                |
-| Resets and Clocks                | SoC should defend against external clock glitching attacks.                                                                                                                                                    | Statement of conformance | Required for Caliptra threat model                |
-| Resets and Clocks                | SoC should defend against external clock overclocking attacks.                                                                                                                                                    | Statement of conformance | Required for Caliptra threat model                |
+| Resets and Clocks                | SoC shall ensure Caliptra clock is derived from an on-die oscillator circuit.                                                                                                                                                                                                  | Statement of conformance | Required for Caliptra threat model                |
+| Resets and Clocks                | SoC shall ensure that any programmable Caliptra clock controls are restricted to the SoC Manager.                                                                                                                                                                              | Statement of conformance | Required for Caliptra threat model                |
+| Resets and Clocks                | SoC should defend against external clock stop attacks.                                                                                                                                                                                                                         | Statement of conformance | Required for Caliptra threat model                |
+| Resets and Clocks                | SoC should defend against external clock glitching attacks.                                                                                                                                                                                                                    | Statement of conformance | Required for Caliptra threat model                |
```

Comment:
`should` implies optionality. Since this is required, update `should`-->`shall`?